src/mixins/common-mixin.ts

```diff
@@ -64,6 +64,13 @@ export class CommonMixin extends Vue {
       case EntityTypes.CR: return CorpTypeCd.BC_COMPANY
       case EntityTypes.UL: return CorpTypeCd.BC_ULC_COMPANY
       case EntityTypes.CP: return CorpTypeCd.COOP
+      case EntityTypes.XUL: return CorpTypeCd.EXTRA_PRO_A
+      case EntityTypes.RLC: return CorpTypeCd.LIMITED_CO
+      case EntityTypes.SO: return CorpTypeCd.SOCIETY
+      case EntityTypes.FR: return CorpTypeCd.SOLE_PROP
+      case EntityTypes.XLP: return CorpTypeCd.XPRO_LIM_PARTNR
+      case EntityTypes.XLL: return CorpTypeCd.XPRO_LL_PARTNR
+      case EntityTypes.XSO: return CorpTypeCd.XPRO_SOCIETY
```

Comment:
Great! But please sort alphabetically to make it easier for a future dev to find things.